app/controllers/secret_shares_controller.rb

```diff
@@ -0,0 +1,9 @@
+class SecretSharesController < ApplicationController
+
+  def create
+    moment = Moment.friendly.find(params[:moments])
```

Comment:
As part of TDD, I would start writing `secret_shares_controller_spec.rb` so that you can test as you go.